tempesta_fw/http_sticky.c

```diff
@@ -316,9 +307,11 @@ tfw_http_sticky_found(TfwHttpMsg *hm, TfwStr *value)
 	/*
 	 * Do nothing for now. The request is passed to a backend server.
 	 */
-	/* XXX This assumes that 'value' is a linear TfwStr{}. */
-	TFW_DBG("Sticky cookie found: \"%.*s\"\n",
-		(int)value->len, (char *)value->ptr);
+	if (likely(TFW_STR_PLAIN(value)))
```

Comment:
Things gets complicated. Maybe, it's better to have general TFW_STR_DBG macro.
Also, an empty `if()` just optimizes out, it should not harm.